pyo3-macros-backend/src/method.rs

```diff
@@ -830,12 +830,14 @@ impl<'a> FnSpec<'a> {
                         _kwargs: *mut #pyo3_path::ffi::PyObject
```

Comment:
I assume these are named with underscores because sometimes the generated code uses them and sometimes it doesn't?